doc/api/util.md

```diff
@@ -426,6 +426,11 @@ passed that alter certain aspects of the formatted string.
 `util.inspect()` will use the constructor's name and/or `@@toStringTag` to make
 an identifiable tag for an inspected value.
 
+Values may supply their own custom inspection function
```

Comment:
As the next code example does not contain any mention of this, maybe we should link `inspection function` here to `#util_util_inspect_custom` or `#util_custom_inspection_functions_on_objects`?

Reply:
Yes, I was also wrapping my head around this. I moved it because when looking at the current documentation it is confusing as it seems to be about something else than it really is. So +1 on that.